Remove systemd dependency because we are install manually the service file.
--- a/cmake/packaging/linux.cmake
+++ b/cmake/packaging/linux.cmake
@@ -8,13 +8,10 @@
     install(FILES "${CMAKE_CURRENT_BINARY_DIR}/sunshine.service"
             DESTINATION "${SUNSHINE_ASSETS_DIR}/systemd/user")
 else()
-    find_package(Systemd)
     find_package(Udev)
 
     install(FILES "${SUNSHINE_SOURCE_ASSETS_DIR}/linux/misc/85-sunshine.rules"
             DESTINATION "${UDEV_RULES_INSTALL_DIR}")
-    install(FILES "${CMAKE_CURRENT_BINARY_DIR}/sunshine.service"
-            DESTINATION "${SYSTEMD_USER_UNIT_INSTALL_DIR}")
 endif()
 
 # Post install
